stable_diffusion.json

```diff
@@ -6871,5 +6631,129 @@
             ]
         },
         "size_on_disk_bytes": 6938043248
+    },
+    "MHXL - Aventis Horizon": {
+        "name": "MHXL - Aventis Horizon",
+        "baseline": "stable_diffusion_xl",
+        "type": "ckpt",
+        "inpainting": false,
+        "description": "Generalist model, focusing on semi-realistic yet stylized generations.",
+        "version": "3",
+        "style": "generalist",
+        "homepage": "https://civitai.com/models/331672/mhxl-aventis-horizon-sdxl",
+        "nsfw": true,
+        "download_all": false,
+        "requirements": {
+            "clip_skip": 2
+        },
+        "config": {
+            "files": [
+                {
+                    "path": "mhxlAventisHorizon_v30.safetensors",
+                    "sha256sum": "BC505CAAB21724E8F4CDEC6AF7A11275BCFC1814AD814806B107474A09B5D7F3"
+                }
+            ],
+            "download": [
+                {
+                    "file_name": "mhxlAventisHorizon_v30.safetensors",
+                    "file_path": "",
+                    "file_url": "https://civitai.com/api/download/models/1026052?type=Model&format=SafeTensor&size=pruned&fp=fp16"
```

Comment:
This does not appear to be the correct link for this model entry; it downloads 'noobevo'. Did you mean https://civitai.com/api/download/models/461878?type=Model&format=SafeTensor&size=pruned&fp=fp16?

Reply:
Thanks for the catch. Yes, I did